src/main/webapp/view/index.html

```diff
@@ -29,7 +29,15 @@ <h2 id="header-text">ZoomTube</h2>
 
   <div id="page">
     <div class="halfpage" id="player-transcript">
-      <div id="player"></div>
+      <div id="video-icons">
+        <div id="player"></div>
+        <div id="icons">
+          <button type="button" onclick="iconOnClick('good')">Thumbs Up</button>
```

Comment:
I think these should be changed to match the capitalization of the `enum`.  `GOOD`, `BAD`, `TOO_FAST`, or whatever they are

Reply:
fixed